app/helpers/application_helper.rb

```diff
@@ -171,15 +167,15 @@ def non_anonymous_text_byline(creation)
   end
 
   def pseud_text(pseud)
-      pseud.byline
+    pseud.byline
   end
 
-   def link_to_modal(content="",options = {})   
+   def link_to_modal(content="",options = {})
      options[:class] ||= ""
      options[:for] ||= ""
      options[:title] ||= options[:for]
-     
-     html_options = {"class" => options[:class] +" modal", "title" => options[:title], "aria-controls" => "#modal"}     
+
+     html_options = {"class" => options[:class] +" modal", "title" => options[:title], "aria-controls" => "#modal"}
```

Comment:
Surrounding space missing for operator '+'.<br>Space inside { missing.<br>Space inside } missing.